tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/view_image.py

```diff
@@ -40,7 +44,7 @@ def tap_back_button(self):
         self.tap_element_from_system_app(header, x=20)
 
         # wait for the frame to close
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != 'Gallery')
+        self.wait_to_not_be_displayed()
```

Comment:
I think this needs to be:
+        from gaiatest.apps.gallery.app import Gallery
+        Gallery(self.marionette).wait_to_be_displayed()